app/factories/attendee.py

```diff
@@ -21,4 +23,4 @@ class Meta:
     is_checked_in = True
     pdf_url = common.url_
     event_id = 1
-    ticket_id = None
+    ticket_id = 1
```

Comment:
revert

Reply:
Can't be reverted. You cannot create an attendee without ticket ID now, and you shouldn't be able to